src/main/java/mods/railcraft/common/worldgen/GeneratorMineZinc.java

```diff
@@ -0,0 +1,23 @@
+package mods.railcraft.common.worldgen;
+
+import mods.railcraft.common.items.Metal;
+import net.minecraftforge.event.terraingen.OreGenEvent.GenerateMinable.EventType;
+
+/**
+ * Created by GeneralCamo on 5/16/2017.
+ *
+ * @author GeneralCamo
+ *         Created for Railcraft <http://www.railcraft.info>
+ */
+public class GeneratorMineZinc extends GeneratorMine {
+
+    private static final int Y_LEVEL = 40;
```

Comment:
I suggest moving this to 30 so it generates on the same level as Lead, they are commonly found together.